src/_shared/scss/_core.scss

```diff
@@ -10,6 +10,13 @@ $gs-column-width: 60px;
 $gs-row-height: 36px;
 $gs-max-columns: 16;
 
+// For popups
+$headerHeight: 36px;
```

Comment:
💥  AND....they're gone https://github.com/guardian/commercial-templates/pull/43/commits/a1b03e865731cf0a8b9aafcbc7803f88c47cdfe7